app/decorators/post_decorator.rb

```diff
@@ -8,7 +8,7 @@ def to_og_params
       locale: 'ja_JP',
       type: 'article',
       title: title,
-      description: render_markdown(body),
+      description: render_markdown(body)&.gsub("\n", ''),
```

Comment:
> By the way using to_s might be meaningful.

:+1: 

It always exists an isomorphism between ""(empty string) and `nil`. (a.k.a. "" and `nil` are invertible.)